apidef/api_definitions.go

```diff
@@ -1284,7 +1284,13 @@ type Provider struct {
 }
 
 type JWTValidation struct {
-	Enabled bool `bson:"enabled" json:"enabled"`
+	Enabled                 bool   `bson:"enabled" json:"enabled"`
+	SigningMethod           string `bson:"signingMethod" json:"signingMethod"`
+	Source                  string `bson:"source" json:"source"`
+	IssuedAtValidationSkew  uint64 `bson:"issuedAtValidationSkew" json:"issuedAtValidationSkew"`
```

Comment:
I think they should be underscore separated i.e. `signing_method` and we should duplicate the same in OAS.

Reply:
okay, so we decide to duplicate them for the sake of consistency